cds/modules/deposit/static/js/cds_deposit/avc/filters/mergeWith.js

```diff
@@ -0,0 +1,8 @@
+function mergeWith() {
+  return function(src1, src2) {
+    return angular.merge({}, src1, src2);
+  }
+}
+
+angular.module('cdsDeposit.filters')
+  .filter('mergeWith', mergeWith);
```

Comment:
I can imagine in 3 months from now we'll not remember what this does, could you please add a small description? :)